ci/do_ci.sh

```diff
@@ -36,6 +38,20 @@ function bazel_debug_binary_build() {
     "${ENVOY_DELIVERY_DIR}"/envoy-debug
 }
 
+function publish_github_release() {
+  TAG=$(git describe --abbrev=0 --tags)
+
+  wget https://github.com/aktau/github-release/releases/download/v0.7.2/linux-amd64-github-release.tar.bz2 -O /tmp/ghrelease.tar.bz2
+  tar -xvjpf /tmp/ghrelease.tar.bz2 -C /tmp
+  cp /tmp/bin/linux/amd64/github-release /usr/local/bin/ghrelease
+  chmod +x /usr/local/bin/ghrelease
+
+  if [[ -n "${TAG:-}" ]]; then
+      ghrelease release --tag "${TAG:-}" --user taion809 --repo envoy-build-tests --name "${TAG:-}"
+      ghrelease upload --tag "${TAG:-}" --user taion809 --repo envoy-build-tests --name "envoy-linux-amd64" --file "${ENVOY_SRCDIR}/build_release_stripped/envoy"
```

Comment:
Does this need to be parameterized somehow to publish to a CNCF user? Looks like this is using your credentials somehow unless I misunderstand it.

Reply:
😱 

yeah it does, that should have been removed.